app/controllers/blacklight/oembed/embed_controller.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module Blacklight::Oembed
+  # Override blacklight-oembed gem controller. We're only interested in
+  # get_embed_content, but keep the changes compatible incase we can use
+  # Blacklight later
+  class EmbedController < ActionController::Base
+    def show
+      render json: { html: get_embed_content(params[:url], additional_params) }
```

Comment:
I think instead of calling these `additional_params`, the rails way of doing is is calling them `options` or `opts`

Reply:
Overridden from: https://github.com/sul-dlss/blacklight-oembed/blob/master/app/controllers/blacklight/oembed/embed_controller.rb

I tried to keep it as backwards compatible as possible in case Hyrax decides to make use of Blacklight show pages in the future. If so, we can reconfigure this to work fairly easily.

Reply:
Cool, can we make a note of that with a comment then? Just for later down the road when no one has looked at the code, we will know this was overridden at some point from somewhere.